### PR TITLE
Digitise ch10 of Infection Prevention and Control

### PR DESCRIPTION
**Pull request by LaurenEllwood**

### CODE REVIEW

File: infection-prevention-and-control/text/10.md [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+Coronaviruses sometimes cross over from an animal species to infect humans. This is believed to be the origin of new human coronaviruses such as SARS-CoV-2.
+
+In this chapter SARS-CoV-2 will be referred to as coronavirus.
+
+Note
+: SARS-CoV-2 is an enveloped single strand RNA virus which needs to enter a living animal cell in order to multiply. When viewed under a microscope the virus looks like a crown with many spikes, hence the name coronavirus. The spike proteins attach to specific receptor sites on the host cell. The viral RNA then enters the mucosal cells of the human hosts’ upper airways, where it multiplies and spreads to other parts of the body.
+
+### 10-3 Where and when did COVID-19 first appear?
+
+In a food market in Wuhan, the capital city of Hubei province in central China during December 2019. In the market a wide range of both dead and alive animals from all over the world were sold. It is believed that the COVID-19 virus crossed from a bat (the primary animal reservoir) to a pangolin (the intermediate host) and finally to a human.
```
Comment (KlaraSkinner) [inline infection-prevention-and-control/text/10.md:63]: "dead and live" rather than "alive"?

File: infection-prevention-and-control/text/10.md [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+- If a person coughs or sneezes into their hand and then touches a surface such as a tap, door handle or phone, coronavirus may be transferred to someone else’s hand if they touch the same surface. If the second person then touches their face (mouth, nose or eyes) coronavirus on their hand may infect their mucous membranes. Coronavirus may survive on a solid surface for 24 hours or longer if not removed by cleaning or disinfection.
+- Coronavirus is also present in the stools of infected people, although it is not currently known how important this route of infection may be. However, the faecal-oral route may also be a method of viral spread.
+
+> Coronavirus is mainly spread by respiratory droplets in the air.Transmission of coronavirus from contaminated surfaces to the face is also possible.
```
Comment (KlaraSkinner) [inline infection-prevention-and-control/text/10.md:80]: Add space after full stop

File: infection-prevention-and-control/text/10.md [hunk 3]
```diff
@@ -0,0 +1,547 @@
...
+
+> Personal protective equipment must be made available for all healthcare workers caring for COVID-19 patients.
+
+### 10-44 Can personal protective equipment be reused?
+
+Due to short supply, PPE often has to be reused. Reuse of PPE (PPE conservation) is still preferable to having no PPE and remains a major challenge all over the world. Aprons and gloves should be replaced between handling patients. Goggles and face shields can be washed clean and disinfected. Paper masks can be used for 8 hours during a work shift but should be discarded if they become moist or get soiled.
+
+### 10-45 For what procedures are N95 masks needed?
+
+N95 masks are only needed by clinical staff when there is a risk of aerosolizing coronavirus i.e. during aerosol generating procedures such as the collection of nasopharyngeal and oropharyngeal swabs for SARS-COV-2 testing, tracheal aspirate, bronchoalveolar lavage, manual bag-mask ventilation, non-invasive CPAP ventilation, tracheal intubation, open suctioning, tracheotomy, bronchoscopy, endoscopy, ENT procedures, dental procedures, maxillo-facial procedures and cardiopulmonary resuscitation. A number of different N95 masks are available.
```
Comment (KlaraSkinner) [inline infection-prevention-and-control/text/10.md:441]: Is a comma needed before "i.e."?

File: infection-prevention-and-control/text/10.md [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+
+### 10-47 How should you handle your uniform or work clothes?
+
+Ideally, used scrubs are laundered at the facility. If this is not possible, remove your work clothes at the workplace and place them in a plastic bag. If removed at home, place them in a washing basket immediately. Wash your hands after removing your clothes. Wash the clothes in hot water if possible (60°C). Clean any spectacles with disinfectant when leaving the COVID-19 ward. Ensure your cell phone has been cleaned.
```
Comment (KlaraSkinner) [inline infection-prevention-and-control/text/10.md:451]: Replace degrees c with unicode symbol

File: infection-prevention-and-control/text/10.md [hunk 5]
```diff
@@ -0,0 +1,547 @@
...
+
+Currently there is no medication proven to kill the coronavirus and cure COVID-19. However, many clinical drug trials show great promise and are being evaluated as potential treatments.
+
+Note
+: The spike proteins of coronavirus attach to ACE2 receptors on the surface of type 2 pneumocytes of the lung which allows the RNA of the virus to enter the cell. Viral proteins and RNA are then produced in the cell to create large numbers of new virus. It is hoped that new treatments will interfere with the attachment and reproduction of coronavirus. The use of commercially produced anti-viral antibodies (coronavirus immunoglobulin) or convalescent serum of people who have recovered from COVID-19 may help patients with severe illness.
+
+### 10-14 How can the clinical diagnosis of COVID-19 be confirmed?
+
+By identifying coronavirus in the upper respiratory tract. This is done by taking a swab from the nasopharyngeal or oropharyngeal and sending it to a laboratory to identify the presence of the coronavirus. This test is reliable in people who are clinically ill and may not detect (a false negative) coronavirus in some people with asymptomatic infection.
```
Comment (LouiseSteward) [inline infection-prevention-and-control/text/10.md:154]: "nasopharyngeal" and "oropharyngeal" are both adjectives, so this sentences needs changing.

File: infection-prevention-and-control/text/10.md [hunk 6]
```diff
@@ -0,0 +1,547 @@
...
+- A plastic apron or a non-sterile long-sleeved gown
+- Non-sterile gloves
+- Face-cover such as a surgical mask for general care and N95 mask (respirator) when indicated
+- Eye protection such as goggles or a face shield.
+
+Donning (putting on) and doffing (taking off) full PPE is a skill that needs practice and must be done correctly to avoid healthcare workers being contaminated with coronavirus.
+
+{% if site.output == "web"%}
+
+View [this demonstration video](https://player.vimeo.com/external/400607941.hd.mp4?s=af075e8c9647a23114424834c1e73f866a73e5f7&profile\_id=174) for safely putting on and taking off PPE.
```
Comment (arthurattwell) [inline infection-prevention-and-control/text/10.md:429]: @LaurenEllwood This video is on Vimeo [here](https://vimeo.com/400607941), which means we can embed it with a `{% include vimeo id="https://vimeo.com/400607941" %}`.

File: infection-prevention-and-control/text/10.md [hunk 7]
```diff
@@ -0,0 +1,547 @@
...
+
+With community lockdown most companies, shops, schools, places of higher learning, meetings, churches, sports clubs and any space where people congregate must be closed. Exceptions include health facilities, family practitioners, a limited number of food stores, banks, fuel stations, and other essential services. This causes severe social and economic disruption, especially for the poor.
+
+With community lockdown (containment) there is a greater chance that public health tracing teams can identify infected individuals, trace and screen their contacts and ensure quarantine of all infected individuals at home or hospital. This task is expensive and requires many health personnel. However, only by preventing the spread of the virus and tracing contacts can the rapid transmission of coronavirus in a community or population be avoided. Failure to do this would be a catastrophe with large numbers of citizens, especially the elderly, dying.
+
+Note
+: The aim of containment or community lockdown is to reduce the reproductive ratio (Ro) i.e. the number of people that an infected person is likely to infect.
```
Comment (LouiseSteward) [inline infection-prevention-and-control/text/10.md:264]: Ro should be R<sub>0</sub> here

**Pull request merged**